tutorials/hosted_phoenix/hosted_phoenix_llamaindex_tutorial.ipynb

```diff
@@ -50,9 +50,11 @@
    "metadata": {},
```

Comment:
<p>Don't need opentelemetry-sdk opentelemetry-exporter-otlp arize-phoenix-otel in the latest versions of phoenix. Just set as &gt;= the version you want</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>